eng/Versions.props

```diff
@@ -89,6 +89,7 @@
     <MicrosoftWindowsCsWin32PackageVersion>0.3.49-beta</MicrosoftWindowsCsWin32PackageVersion>
     <!-- This is the version of the zip archive for the WiX toolset and is different from the NuGet package version format. -->
     <WixVersion>3.14.1.8722</WixVersion>
+    <MicrosoftSbomTargetsPackageVersion>0.0.0-preview.0</MicrosoftSbomTargetsPackageVersion>
```

Comment:
This will be updated once we publish an official Nuget package.

- [ ] Update this with the official version.